### ISSUE TITLE
Make property proposal for Web Accessibility Directive report URL

### ISSUE THREAD

**Issue by Ainali** [Ainali/ideas#32]
https://www.wikidata.org/wiki/Q96414359

**Comment by Ainali** [comment]
Planned for tomorrow's Live Wikidata editing: https://www.youtube.com/watch?v=hFUzh8U9Bl8

**Comment by Ainali** [comment]
Perhaps this should be wider also, for accessibility reports in general, and then perhaps qualified with https://www.wikidata.org/wiki/Property:P5009

**Comment by Ainali** [comment]
Property created: https://www.wikidata.org/wiki/Property:P9494
